examples/eks-cni-example.jsonnet

```diff
@@ -2,8 +2,6 @@ local kp = (import 'kube-prometheus/main.libsonnet') + {
   values+:: {
     common+: {
       namespace: 'monitoring',
-    },
-    kubePrometheus+: {
       platform: 'eks',
     },
   },
```

Comment:
You might want to clean up `prometheusRuleEksCNI` below, it does not look valid too me, and maybe add an example to configure the CNI thresholds like:
```jsonnet
    awsVpcCni: {
      minimumWarmIPs: 10,
    },
```